### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by IonatanSala**
Created static pages , terms, about us and privacy policy.

### CODE REVIEW

**Comment by danielpaul** [comment]
@IonatanSala Why was the `schema.rb` file modified in this branch?

**Comment by danielpaul** [comment]
@IonatanSala You should use https://github.com/thoughtbot/high_voltage for the static pages. Had added it in the description for the card in Trello. You don't need to declare the routes and methods like you do here for static pages.

**Comment by IonatanSala** [comment]
Not really sure what happend with the schema.rb file here, I don't recall touching the database files.

**Comment by IonatanSala** [comment]
I'll have a look at that gem in the morning.

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -9,4 +9,16 @@ def dashboard
     @purchases = current_user.purchases.all
   end
 
+  def terms_and_conditions
```
Comment (danielpaul) [inline app/controllers/pages_controller.rb]: You can delete these methods @starkshaw

**Comment by starkshaw** [comment]
Needs review, not sure what else needs to be done.

File: app/views/pages/about_us.html.haml [hunk 2]
```diff
@@ -0,0 +1 @@
+%h3 About Us
```
Comment (danielpaul) [inline app/views/pages/about_us.html.haml:1]: Could we add some dummy text on all these pages?

File: db/schema.rb [hunk 3]
```diff
@@ -25,6 +25,16 @@
     t.datetime "updated_at"
   end
 
+  create_table "cards", force: :cascade do |t|
```
Comment (danielpaul) [inline db/schema.rb]: Why are these modified again? The schema should not have been in this at all.

Comment (starkshaw) [inline db/schema.rb]: I don't know...

Comment (starkshaw) [inline db/schema.rb]: This is bullshit. I will copy the `schema.rb` from `master`.
